portfolio/src/main/webapp/js/project-detail.js

```diff
@@ -40,9 +40,26 @@ for(let detail of project.detail){
      /** @type {CommentData} */
      const comments = await response.json();
 
+    /** Only show delete comments button if comments exist. */
+    const deleteCommentsFormNode = document.querySelector("#delete-comments");
+    if(comments.length > 0){
+        deleteCommentsFormNode.classList.remove("hidden");
+    } else {
+        deleteCommentsFormNode.classList.add("hidden");
+    }
+
      Comment.populateAll(comments);
  }
 
+/** Delete all comments for this project, then reload. */
+async function deleteComments(){
+    await await fetch(`/comments?projectId=${projectId}`, {
```

Comment:
await await?

Reply:
Oops. Typo that would be caught in any other language, but runs fine in javascript :) Fixed!